ddtrace/contrib/internal/botocore/services/bedrock.py

```diff
@@ -318,7 +318,11 @@ def handle_bedrock_response(
 def patched_bedrock_api_call(original_func, instance, args, kwargs, function_vars):
     params = function_vars.get("params")
     pin = function_vars.get("pin")
-    model_provider, model_name = params.get("modelId").split(".")
+    model_meta = params.get("modelId").split(".")
+    if len(model_meta) == 2:
+        model_provider, model_name = model_meta
+    else:
+        _, model_provider, model_name = model_meta  # cross-region inference
```

Comment:
Approving to unblock the fix but should we do something like
```
if len(model_meta) == 2:

elif len(model_meta) == 3

else:
   log.warning('failed to parse model name')
   model_name = placeholder
   model_provider = placeholder
```
in a follow up PR?

Reply:
ah hmm yeah, good question. i'm not sure what the standard is in general for un-parsable things we want to tag (not sure how common placeholders are, and if they serve any value), and i also don't think logging from an integration itself is standard. we'll follow-up on this next week 😄

Reply:
and for this particular case, i think it can only be `provider.model` or `region.provider.model` ([ref](https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids.html), [ref](https://docs.aws.amazon.com/bedrock/latest/userguide/cross-region-inference-support.html))

Reply:
resolving for now, will follow up this week